orchestra/analytics/latency.py

```diff
@@ -128,3 +132,55 @@ def project_time_row_generator(project_information,
             [iteration.end_datetime for iteration in iterations])
         for start_datetime, iteration in zip(iteration_times[:-1], iterations):
             yield iteration.to_dict(start_datetime)
+
+
+def work_time_aggregate(df, dimensions):
+    """
+    Aggregates work time grouped by `dimensions`.
+
+    Args:
+        df (pandas.DataFrame):
+            A DataFrame with a schema described in `work_time_df`.
+        dimensions (list):
+            A list of column names to group on.
+
+    Returns:
+        df (pandas.DataFrame):
+            A DataFrame with aggregate information.
+    """
+    return df.groupby(dimensions, as_index=False).agg({'work_time': np.sum})
```

Comment:
* Is it Orchestra's job to provide analytics convenience methods?
* This is a really thin wrapper around pandas—how useful is it?
* what does `as_index` do?